workflow/controller/operator.go

```diff
@@ -140,6 +143,20 @@ type failedNodeStatus struct {
 
 // newWorkflowOperationCtx creates and initializes a new wfOperationCtx object.
 func newWorkflowOperationCtx(wf *wfv1.Workflow, wfc *WorkflowController) *wfOperationCtx {
+	var rateLimiter *waitutil.RateLimiter
+	rateLimiter = nil
```

Comment:
Do we need to set this to nil?

Reply:
it should be nil already but I think being more explicit is better ?

Reply:
I think this makes it harder to read since the default value is nil. But if we really want to set it explicit I suggest: 

```
var rateLimiter *waitutil.RateLimiter = nil 
```

Reply:
addressed